ansible-ipi-install/roles/node-prep/tasks/10_validation.yml

```diff
@@ -38,9 +38,11 @@
   tags: validation
 
 - name: Verify DNS records for API VIP, Wildcard (Ingress) VIP
+  vars:
+    qtype: "{{ 'qtype=AAAA' if ipv6deployment is defined and ipv6deployment|bool else 'qtype=A' }}"
```

Comment:
Lets move this qtype variable into roles/node-prep/vars/main.yml

Reply:
iiuc that the purpose of `bool` filter. Workd for me locally with both `ipv6_enabled=true` and `ipv6_enabled=True`

Reply:
awesome! I actually didn't know that.